flexget/plugins/api_tvdb.py

```diff
@@ -539,6 +593,45 @@ def lookup_episode(name=None, season_number=None, episode_number=None, absolute_
         raise LookupError('No results found for %s' % ep_description)
 
 
+@with_session
+def search_for_series(series_name=None, imdb_id=None, zap2it_id=None, force_search=None, session=None):
+    """
+    Search IMDB using a an identifier, return a list of cached search results. One if `search_name`, `imdb_id` or
+    `zap2it_id` is required.
+    :param series_name: Name of search to use
+    :param imdb_id: Search via IMDB ID
+    :param zap2it_id: Search via zap2it ID
+    :param force_search: Should search use cache
```

Comment:
I had to read this a couple of times to understand that it means "Should the search use cache?"